CODEOWNERS

```diff
@@ -0,0 +1,3 @@
+# Order is important. The last matching pattern has the most precedence.
+
+* @rancher/fleet
```

Comment:
That's a good point, I like the CODEOWNER because it increases ownership but it can also lead to bystander effect :/
So I am fine to close this PR if you think it won't help

Reply:
I think we should discuss it in our next meeting. From there we can make a decision.

Reply:
We discussed this topic in our weekly meeting and we agree that the main benefit that CODEOWNER brings to the project is increasing repository ownership. The goal is to help external contributors to identify who to contact for PR validation or to ask questions to the fleet team, in issue/pr comments by using the word `@rancher/fleet `  

At this time we are not interested to enforce pullrequest validation based on CODEOWNER, nor assigning automatically pullrequest to rancher/fleet, even though the later is a consequence that we can avoid.